src/firefly/java/edu/caltech/ipac/firefly/server/query/IrsaLightCurveHandler.java

```diff
@@ -0,0 +1,121 @@
+/*
+ * License information at https://github.com/Caltech-IPAC/firefly/blob/master/License.txt
+ */
+package edu.caltech.ipac.firefly.server.query;
+
+import edu.caltech.ipac.astro.IpacTableWriter;
+import edu.caltech.ipac.firefly.server.ServerContext;
+import edu.caltech.ipac.util.DataGroup;
+import edu.caltech.ipac.util.VoTableUtil;
+import edu.caltech.ipac.util.download.FailedRequestException;
+import edu.caltech.ipac.util.download.URLDownload;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URL;
+import java.net.URLConnection;
+
+
+/**
+ * .
+ * Should handle the LC transformations to get files out of the API result VOtable xml
+ *
+ * @author ejoliet
+ * @see PeriodogramAPIRequest
+ */
+public class IrsaLightCurveHandler extends NexsciLcApiHandler {
```

Comment:
Is there a reason for subclassing?  I don't see multiple versions of nexsci handlers.